api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -417,6 +429,15 @@ def __attrs_post_init__(self):  # noqa: C901
         if self.source == 'local':
             if not self.local_repo:
                 raise ValueError("local repo is undefined")
+            if (
+                not self.disable_pip_repo and
```

Comment:
As per the comment above that options meaning is not clear. So this and similar conditionals are not clear as well. Please clarify.

Reply:
Clarified in above comment
https://github.com/Seagate/cortx-prvsnr/pull/681#discussion_r513377085